dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/event/ProgramMessageController.java

```diff
@@ -126,10 +128,16 @@ public List<ProgramMessage> getProgramMessages(
   @GetMapping(value = "/scheduled/sent", produces = APPLICATION_JSON_VALUE)
   @ResponseBody
   public List<ProgramMessage> getScheduledSentMessage(
-      @Deprecated(since = "2.41") @RequestParam(required = false) UID programInstance,
-      @RequestParam(required = false) UID enrollment,
-      @Deprecated(since = "2.41") @RequestParam(required = false) UID programStageInstance,
-      @RequestParam(required = false) UID event,
+      @OpenApi.Param(value = Enrollment.class)
```

Comment:
I would expect `@OpenApi.Param({UID.class, ProgramInstance.class})` or something like that